spec/rspec/active_job/enqueue_a_spec.rb

```diff
@@ -148,4 +148,31 @@ class BJob; end
       end
     end
   end
+
+  context "with run time expectations" do
+    let(:instance) { described_class.new }
+    let(:run_time) { Time.parse('2015-09-10').to_f }
+    subject(:matches?) { instance.to_run_at(run_time).matches?(AJob) }
+
+    let(:enqueued_jobs) do
+      [{ job: AJob, args: [], at: time }]
+    end
+
+    context "correct time" do
+      let(:time) { run_time }
+      it { is_expected.to be(true) }
+    end
+
+    context "wrong time" do
+      let(:time) { run_time + 1 }
+      it { is_expected.to be(false) }
+
+      specify do
+        matches?
+        expect(instance.failure_message).
+          to eq("expected to run job at 2015-09-10 00:00:00 +0300, " \
```

Comment:
When I run this spec locally I get a failure because I'm in BST (+0100), so the output strings don't match. Could we pop `.utc` on `run_time` in here, and in `format_enqueued_times` and `failure_message`, for consistency?